packages/loot-core/webpack/webpack.api.config.js

```diff
@@ -15,6 +15,7 @@ config.resolve.extensions = [
   '.json',
 ];
 config.resolve.fallback = {};
+config.externals.push('@actual-app/crdt');
```

Comment:
Not necessary, but if we are going to depend on the type package in the api, might as well use it rather than have it in the webpack bundle.  I did confirm the built API was still working after this change.